### ISSUE TITLE
Enable incremental package builds for RPM and DEB - on dev machines

### ISSUE THREAD

**Issue by NikolaMilosavljevic** [dotnet/runtime#32304]
It would be great to build binaries only once, and use appropriate DEB and RPM platforms to build packages from the same portable artifacts drop.

For instance, you should be able to do the following:
1) Build binaries using CentOS image (for instance `mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-6aaa05d-20191106231356`)
2) Add FPM tooling needed for RPM packaging and build RPM packages
3) Switch to Ubuntu container (i.e. `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-debpkg-cfdd435-20200121150040`) and build just DEB packages (by using `installer` subcategory)

This process is currently not possible due to some cached properties. `HostMachineRid` is built during full build only (using `tools-local/tasks/installer.tasks/installer.tasks.csproj`) and cached in `artifacts/obj/HostMachineInfo.props`. Subsequent partial builds would use the cached `HostMachineRid` value.

**Comment by ViktorHofer** [comment]
> This process is currently not possible due to some cached properties. HostMachineRid is built during full build only (using tools-local/tasks/installer.tasks/installer.tasks.csproj) and cached in artifacts/obj/HostMachineInfo.props. Subsequent partial builds would use the cached HostMachineRid value.

@NikolaMilosavljevic there is no HostMachineInfo.props anymore. Can you please verify if the desired workflow is still not possible on master?

**Comment by NikolaMilosavljevic** [comment]
> @NikolaMilosavljevic there is no HostMachineInfo.props anymore. Can you please verify if the desired workflow is still not possible on master?

Thanks @ViktorHofer - I'll verify if this is still an issue.

**Comment by NikolaMilosavljevic** [comment]
[Triage] Not a priority.
